desktop/models/sale.coffee

```diff
@@ -39,6 +39,8 @@ module.exports = class Sale extends Backbone.Model
 
   bidUrl: (artwork) ->
     "#{@href()}/bid/#{artwork.id}"
+  
+  predictionUrl: -> "#{process.env.PREDICTION_URL}/#{@get('slug')}"
```

Comment:
I just noticed that jade templates in other areas do something like
`#{sd.PREDICTION_URL}/#{auction.get('id')}#{user ? '/login' : ''}"`. I need to manually test my code anyways so I'll verify this.